deps/rabbitmq_management/src/rabbit_mgmt_wm_operator_policy.erl

```diff
@@ -72,7 +72,12 @@ delete_resource(ReqData, Context = #context{user = #user{username = Username}})
     {true, ReqData, Context}.
 
 is_authorized(ReqData, Context) ->
-    rabbit_mgmt_util:is_authorized_admin(ReqData, Context).
+    case rabbit_mgmt_util:is_edit_op_policy_disabled() of
```

Comment:
I think such restrictions should live in a separate module, r.g. `rabbit_mgmt_features`.